Week-2/Html Basics/createList.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>List</title>
+</head>
+<body>
+    <ol>
```

Comment:
This code has structural issues. In HTML,` <ul>` (unordered list) and `<ol>` (ordered list) elements should contain `<li>` (list item) elements directly as their children. You should not place a `<ul>` or `<ol>` directly within another `<ol>`. Instead, you can use nested lists by placing one list within a list item of another list. Here's how you can correct the code:

```
<ol>
    <li>Desktop
        <ul>
            <li>Linux</li>
            <li>Mac OS</li>
            <li>Windows</li>
        </ul>
    </li>
    <li>Mobile
        <ul>
            <li>Android</li>
            <li>iOS</li>
        </ul>
    </li>
</ol>
```